packages/pangea-sdk/tests/integration/asyncio/test_authn.py

```diff
@@ -279,12 +279,12 @@ async def authn_c2_login_n_client_session_invalidate(self):
             for session in response.result.sessions:
                 try:
                     await self.authn.client.session.invalidate(token=token, session_id=session.id)
-                except pexc.PangeaAPIException as e:
+                except pe.PangeaAPIException as e:
                     print(f"Fail to invalidate session_id[{session.id}] token[{token}]")
```

Comment:
This should be "Failed to invalidated..."

Reply:
fixed